scripts/wasmd/deploy_cw3.js

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env node
+
+/* eslint-disable @typescript-eslint/naming-convention */
+const { DirectSecp256k1HdWallet } = require("@cosmjs/proto-signing");
+const { SigningCosmWasmClient } = require("@cosmjs/cosmwasm-stargate");
+const fs = require("fs");
+
+const endpoint = "http://localhost:26659";
+const alice = {
+  mnemonic: "enlist hip relief stomach skate base shallow young switch frequent cry park",
+  address0: "wasm14qemq0vw6y3gc3u3e0aty2e764u4gs5lndxgyk",
+  address1: "wasm1hhg2rlu9jscacku2wwckws7932qqqu8xm5ca8y",
+  address2: "wasm1xv9tklw7d82sezh9haa573wufgy59vmwnxhnsl",
+  address3: "wasm17yg9mssjenmc3jkqth6ulcwj9cxujrxxg9nmzk",
+  address4: "wasm1f7j7ryulwjfe9ljplvhtcaxa6wqgula3nh873j",
+};
+
+const codeMeta = {
+  source: "https://crates.io/api/v1/crates/cw3-fixed-multisig/0.3.1/download",
```

Comment:
Nice. There is a v0.4.0 coming out in the next week or two with plenty of changes to multisigs and groups.

I think minimal breaking API changes, but a lot more power.